domains/neerav.json

```diff
@@ -0,0 +1,14 @@
+{
+        "owner": {
+           "username": "BadPotato1007",
+           "email": "",
+           "discord": "1176835313356767312",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.h5lVmEVgNjSJED4_lCqP0uq8U6PetLRaVdYIsrVbDslLhV0Nmvkdj9AbxxT1TGf3ZGwFSSJe-0zei3AeiQ-opshMYrw_DRlx4f686t-AhNXj9Kmf2-0wQxjsmUvwXNsmNfgwxjXtD5gHv8jLOF1oh_ZV9rbjCmWUejPLX6a0RvRk-hH9ACxziJ3qzdY88SdlXcqA6KqxhD0ZDyWt_NAXk-DIDv-oOoeX-MM9ZW1f89HDXvQPZ3uqOmcdzJtoCBGGAt4nY9C0fkK0YGSp_lXe_r6G7gjhNfwEH_BQ997SpDpibdH3ensKeGf6DOufxaN1FK8L0KCf-XpLXELc9c2qUw.bEd83RTDvTOZ19pDJ6YrTA.hj0EVK9MR1_ygJEtgrhofZrU37pBoHu-zXlTGQSG0DuRAi1VQC3jOP6QrMJZi7zWZbce2Y83elAYzq_I4W0KrshAwfAUlcr6DRTb5Hl4JdU.RFRNhW4hhRsoe9Mw-F-fwg"
+        },
+    
+        "record": {
+            "CNAME": "apex-loadbalancer.netlify.com",
+            "A": "75.2.60.5"
```

Comment:
Cannot have CNAME records in conjunction with other records.

Reply:
Done